### ISSUE TITLE
[REQUEST] [open-ce-builder] - --run_tests option should be made default 

### ISSUE THREAD

**Issue by npanpaliya** [open-ce/open-ce#566]
To test the package updated we should enable `--run_tests` by default. This will make sure every change we do is verified just after the build.

**Comment by npanpaliya** [comment]
For this we need to create short tests in feedstocks' open-ce-tests.yaml that don't take more time to run, else the build time would increase. We need to brainstorm more on this how to have short/long tests in open-ce-tests.yaml.

**Comment by cdeepali** [comment]
Yes I agree, this just requires a minimal test to keep the build times unaffected.
